TestCases/compliance-level-3/0082-feel-coercion/0082-feel-coercion.dmn

```diff
@@ -147,8 +147,8 @@
     </decision>
 
     <businessKnowledgeModel name="bkm_004" id="_bkm_004">
-        <variable typeRef="tNumberList" name="bkm_004"/>
-        <encapsulatedLogic>
+        <variable name="bkm_004"/>
+        <encapsulatedLogic typeRef="tNumberList">
```

Comment:
Hi @opatrascoiu - as per the comments on bkm_005.  The positioning of the type is part of the intention of the test. Though, I see the test for it is commented out referencing this: https://github.com/dmn-tck/tck/pull/238#issuecomment-497744315.  I think that was a back-office change and not by me.  It is maybe possible the intention of some tests that comment refers to may also have been misunderstood.   Unsure.